### DISCUSSION TITLE
Drawer Service: How to prevent the drawer from opening and closing on key presses (Space key and Enter key)?

### DISCUSSION THREAD

**Discussion by Rakshana-Sri** [NG-ZORRO/ng-zorro-antd#8945] [Q&A]
When the drawer is opened, pressing the space key or enter key closes it. Similarly, pressing the space key or enter key reopens it. How can I prevent this behaviour?

**Environment | Info**
ng-zorro-antd - 18.2.1
Browser - Chorme

**Comment by ParsaArvanehPA** [comment]
Hi @Rakshana-Sri ,
Drawer has a `[nzKeyboard]` property to manage this exact behavior.
Try setting it to false.

https://ng.ant.design/components/drawer/en#nz-drawer
